downstream/modules/platform/proc-controller-manage-instances.adoc

```diff
@@ -0,0 +1,83 @@
+[id="proc-controller-manage-instances"]
+
+= Managing instances
+
+Use the following procedure to manage instances:
```

Comment:
Adheres to our usual standard of punctuation before the sub-heading, **Procedure**.

```suggestion
Use the following procedure to manage instances.
```